packages/flutter_tools/lib/executable.dart

```diff
@@ -111,9 +111,11 @@ Future<Null> main(List<String> args) async {
         stderr.writeln(chain.terse.toString());
         stderr.writeln();
       }
-      stderr.writeln('If this problem persists, please report the problem at');
-      stderr.writeln('https://github.com/flutter/flutter/issues/new');
-      _exit(error.exitCode ?? 65);
+      if (error.isUnusual) {
+        stderr.writeln('If this problem persists, please report the problem at');
```

Comment:
While we're here, let's remove the "if it persists". We want all crash reports, not just the bad ones.